app/assets/javascripts/application.js

```diff
@@ -21,12 +20,12 @@ var Hours = Hours || {
   }
 };
 
-$('.datepicker').each(function () {
-  new Pikaday({
-    field: this,
-    format: I18n[$('body').data('language')].date.format
-  });
-});
+// $('.datepicker').each(function () {
+//   new Pikaday({
+//     field: this,
+//     format: I18n[$('body').data('language')].date.format
+//   });
+// });
```

Comment:
sorry removed it cause it was failing at first. due sprockets error. will change :)